### ISSUE TITLE
Remove any downstream code that expects to have "organoid" or "cell culture" In the name of the tissue

### ISSUE THREAD

**Issue by joyceyan** [chanzuckerberg/single-cell-data-portal#6076]

**Comment by prathapsridharan** [comment]
@joyceyan @nayib-jose-gloria  @dsadgat  - I think it is best to do this after the migration is successful. If for some reason we hit a roadblock where we can't migrate to schema-4 and must stay in schema-3 then the schema-3 compatible test fixtures will be needed to verify that the code is compatible with schema-3 formats.

If y'all agree, I can remove this ticket from this sprint

**Comment by joyceyan** [comment]
Yeah, I think that's a good point

**Comment by prathapsridharan** [comment]
@joyceyan - Is this worth doing manually? Seems highly error prone. Perhaps it is best to regenerate these files using schema-4. Especially, if we [remove suffix logic as in this ticket](https://github.com/chanzuckerberg/single-cell/issues/525), I am not sure it is worth the risk of manually removing these strings.

The risk is that some files are simply lists of strings (which is easy enough). Others are data structures that represent hierarchy relationships. The risk is messing up the structure of a file by doing this manually.

**Comment by joyceyan** [comment]
@prathapsridharan Yeah I would expect this to not be done manually, we should regenerate the test files in the same manner they were originally generated.

**Comment by prathapsridharan** [comment]
@joyceyan - Since I am working on census now, I am removing myself from this ticket.

**Comment by joyceyan** [comment]
@dsadgat Should I / Data Portal pick this up or is there someone on data vis who can take this on?

**Comment by brianraymor** [comment]
Removed from [Increase the velocity of dataset schema evolution and dataset migration](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell/365) because it's P2 and blocking the closure of the epic. CC: @dsadgat
